components/greaselion/browser/greaselion_service_impl.cc

```diff
@@ -249,7 +254,8 @@ void GreaselionServiceImpl::CreateAndInstallExtensions() {
   std::vector<std::unique_ptr<GreaselionRule>>* rules =
       download_service_->rules();
   for (const std::unique_ptr<GreaselionRule>& rule : *rules) {
-    if (rule->Matches(state_) && rule->has_unknown_preconditions() == false) {
+    if (rule->Matches(state_, browser_version_) &&
+        rule->has_unknown_preconditions() == false) {
```

Comment:
Nit: `!rule->has_unknown_preconditions()` is a little more idiomatic.

Reply:
I changed this in https://github.com/brave/brave-core/pull/6843